src/main/groovy/edu/wpi/first/gradlerio/wpi/dependencies/tools/WPIToolsPlugin.groovy

```diff
@@ -21,10 +21,10 @@ class WPIToolsPlugin implements Plugin<Project> {
             String toolFolder = new File(frcHome, 'tools').toString()
 
             ToolInstallTask.toolsFolder = toolFolder
-            tools << new WPITool(project, "SmartDashboard", wpi.smartDashboardVersion, "edu.wpi.first.wpilib:SmartDashboard:${wpi.smartDashboardVersion}", false)
+            tools << new WPITool(project, "SmartDashboard", wpi.smartDashboardVersion, "edu.wpi.first.tools:SmartDashboard:${wpi.smartDashboardVersion}", true)
             tools << new WPITool(project, "ShuffleBoard", wpi.shuffleboardVersion, "edu.wpi.first.shuffleboard:shuffleboard:${wpi.shuffleboardVersion}", true)
             tools << new WPITool(project, "OutlineViewer", wpi.outlineViewerVersion, "edu.wpi.first.wpilib:OutlineViewer:${wpi.outlineViewerVersion}", true)
```

Comment:
Out of interest; why are SmartDashboard and RobotBuilder in a `tools` 'namespace' and PathWeaver, OutlineViewer are in the main `wpilib` namespace (and Shuffleboard is in its own `shuffleboard` namespace)? 
Shouldn't they be in similar artifact locations, given that they're all tools? Unless there is some difference I'm not seeing.

Reply:
We're in the middle of moving the tools to `tools`. OutlineViewer just hasn't been merge yet, although Shuffleboard won't be moving until 2022 because of its plugin setup.

Reply:
And I assume the other is a simple copy-pasta mistake?